test/test-install-same.conf

```diff
@@ -0,0 +1,45 @@
+# testsuite system configuration
```

Comment:
This should perhaps be named `test/test-install-same-false.conf`

Reply:
Will rename